daemon/api.go

```diff
@@ -803,6 +828,18 @@ func trySnap(c *Command, r *http.Request, user *auth.UserState, trydir string, f
 	return AsyncResponse(nil, &Meta{Change: chg.ID()})
 }
 
+func isTrue(value []string) bool {
```

Comment:
Sorry, I know this is bike-shedding and please do not block on it. However: `isTrue([]string{"false", "true"}) == false` makes me uneasy. Maybe `func formValueTrue(form *multipart.Form, field string) bool` ? or even `isTrue(form,field)` just something that is less generic than []string while looking like its very generic :)